modules/standard/AutoGpu.chpl

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pragma "module included by default"
+@unstable("The module name 'AutoGpu' is unstable.")
```

Comment:
As this is an auto-included module and we already have an unstable warning for the GPU locale model, I think this should be removed.

Reply:
I believe this warning only flags if you explicitly use `AutoGpu`; the same is true for `AutoMath`, which is where I got this from. Hence, the module _name_ is unstable.